packages/picasso.js/types/index.d.ts

```diff
@@ -0,0 +1,1694 @@
+declare const pjs: picassojs;

```

Comment:
It would have been nice with a disclaimer at the top of this file stating that it's autogenerated and should not be touched.

Reply:
Good idea. The file is regenerated when running `yarn ts` so it needs to be added to `@scriptappy/to-dts`. Optionally, a flag can added to the CLI options controlling if it should be included.

Reply:
Probably better to add it as an optional flag and disable it by default. To avoid introducing a change for anyone already using it.